.github/workflows/ci-cd.yml

```diff
@@ -127,12 +124,36 @@ jobs:
         with:
```

Comment:
Also, it looks like that comment a few lines above is no longer true.

Reply:
Fixed stale comment in 0e43141d7c5d85381dea34933b73b84f00da798c